bin.js

```diff
@@ -1,23 +1,23 @@
 #!/usr/bin/env node
 
-const program = require('commander')
-  .version(require('./package.json').version)
-  .usage('[options] <files>')
-  .parse(process.argv)
+const program = require("commander")
```

Comment:
are we using a linter for this?

Reply:
No, want me to add eslint and prettier standard?... with commas, etc?

Reply:
style changes are only valid if it is automated via a linter/prettier :)

Reply:
My editor defaults did all that ish on save.